src/almanack/metrics/data.py

```diff
@@ -166,6 +166,26 @@ def is_citable(repo: pygit2.Repository) -> bool:
     return False
 
 
+def default_branch_is_not_master(repo: pygit2.Repository) -> bool:
+    """
+    Checks if the default branch of the specified
+    repository is "master".
+
+    Args:
+        repo (Repository):
+            A pygit2.Repository object representing the Git repository.
+
+    Returns:
+        bool:
+            True if the default branch is "master", False otherwise.
+    """
+    # Get the symbolic reference to HEAD, which should point to the default branch
+    head_ref = repo.head.shorthand
+
+    # Check if the default branch is not "master"
+    return head_ref != "master"
```

Comment:
don't most (and now default) repos use `main`? I'm wondering how this check might change in the future as more and more repos use main

Reply:
`git` by default still references `master`. New settings were enabled to allow for a global override sometime in 2020 ([see here](https://github.blog/open-source/git/highlights-from-git-2-28/)) but `master` is still the embedded default without this being set. I double checked with this quick Dockerfile example https://gist.github.com/d33bs/dc9d2a8f64b792b73478e124db3e8601 and I see `master` when I run the container. There is a warning that the default is subject to change, yet it persists for now.